daprdocs/content/en/reference/components-reference/supported-secret-stores/_index.md

```diff
@@ -9,6 +9,8 @@ aliases:
 no_list: true
 ---
 
+The following table lists secret stores supported, at various levels, by the Dapr secrets building block. [Learn how to set up different secret stores for Dapr secrets management.]({{< ref setup-secret-store.md >}})
```

Comment:
Same comment as above. What did you mean by "at various levels"? Do you mean certification status?